app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import random
+
+from dateutil.parser import isoparse
+
+from django.contrib.gis.geos import Polygon
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import Item
+from stac_api.validators import MEDIA_TYPES
+
+# path definition relative to the directory that contains manage.py
+logger = logging.getLogger(__name__)
+
+XMIN = 2570000
+# this would be the max extent for CH
+# XMAX = 2746000
+XMAX = 2580000
+YMIN = 1130000
+# this would be the max extent for CH
+# YMAX = 1252000
+YMAX = 1140000
+
+MIN_DATETIME = isoparse('1970-01-01T00:00:01Z')
+MAX_DATETIME = isoparse('2020-12-31T23:59:10Z')
+
+
+def random_datetime(start, end):
+    """Generate a random datetime between `start` and `end`"""
+    return start + datetime.timedelta(
+        # Get a random amount of seconds between `start` and `end`
+        seconds=random.randint(0, int((end - start).total_seconds())),
+    )
+
+
+class Command(BaseCommand):
```

Comment:
Measuring the execution time of this command and printing (using `print()` not logger) it at the end might be a good idea. This way we could see how our code change impact the DB creation performance by checking out earlier version and executing this command

Reply:
I know that this would not give a reliable measure, but more kind of a hint. I mean if suddenly it takes twice the time or even more, then we might have an issue, but small times difference it won't help.